src/exercicio-01/main.ts

```diff
@@ -0,0 +1,35 @@
+// 1 - Criar uma função que retorne a quantidade de vogais da palavra passada.
+// a) Dar um exemplo de uso com uma palavra recebida via parâmetro da função.
+// b) Dar um exemplo de uso com uma palavra recebida via input no formulário.
+
+function contarVogais(palavra: string): number {
+  const vogais = "aeiouAEIOU";
```

Comment:
Podes verificar para vogais com acento também e não precisas acrescentar as vogais maiúsculas, podes forçar tudo para minúsculo com toLowerCase